README.md

```diff
@@ -155,22 +155,28 @@ main()
 You can add custom labels to your traces. Filters can later be used for filtering
 traces that contains specific labels:
 ```python
-@epsagon.lambda_wrapper
 def handler(event, context):
-  epsagon.label('label', 'something_to_filter_afterwards')
-  epsagon.label('number_of_records_parsed_successfully', 42)
-  pass
+    epsagon.label('key', 'value')
+    epsagon.label('user_id', event['headers']['auth'])
+    epsagon.label('number_of_records_parsed_successfully', 42)
 ```
 
 ### Custom Errors
 
-Set a custom error, maybe without even failing the function:
+Set a custom error, even if handled correctly:
```

Comment:
I think we need a bit more verbose documentation here.
Right now the docs don't mention anything about how we aggregate issues
(all different "string" set_errors will be aggregated to the same issue as far as I remember)

Reply:
I agree, I put a reference here, because it is not the right place to document product behaviour